examples/prefect_example_cmd.sh

```
@@ -0,0 +1 @@
+prefect deployment run 'FreeEnergyLineWorkflow/dpti-workflow-line-deployment' -p flow_trigger_dir=$(pwd)
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)